juicy-html.html

```diff
@@ -36,14 +38,26 @@
             }
         };
         /**
+         * Skips/disregards pending request if any
+         * @return {XMLHttpRequest} skipped XHR request, in fact just `this.pending`
+         */
+        JuicyHTMLPrototype.skip = function() {
+            if(this.pending){
+                this.pending.onload = null;
```

Comment:
I did. But since we cannot even mimic such abort with HTML imports, I'd rather keep both element consistent.